packages/web/src/scss/components/Dropdown/index.html

```diff
@@ -1,30 +1,297 @@
 {{#> layout/plain }}
 
-<div class="Dropdown is-open">
-  <a href="#" class="d-flex mb-400">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#info" />
-    </svg>
-    <span>Information</span>
-  </a>
-  <a href="#" class="d-flex mb-400">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#link" />
-    </svg>
-    <span>More links</span>
-  </a>
-  <a href="#" class="d-flex mb-400">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#profile" />
-    </svg>
-    <span>Profile</span>
-  </a>
-  <a href="#" class="d-flex">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#help" />
-    </svg>
-    <span>Help</span>
-  </a>
-</div>
+<section class="docs-Section">
+
+  <h2 class="docs-Heading">Default usage</h2>
+
+  <div class="DropdownWrapper">
+    <button
+      data-toggle="dropdown"
+      data-target="#DropdownDemo1"
+      class="Button Button--primary Button--medium"
+      aria-expanded="false"
+      aria-controls="#DropdownDemo1"
+    >
+      Button as anchor
+    </button>
+    <div class="Dropdown Dropdown--bottom Dropdown--left" id="DropdownDemo1">
+      <a href="#" class="d-flex mb-400">
+        <svg width="24" height="24" aria-hidden="true" class="mr-400">
+          <use xlink:href="/icons/svg/sprite.svg#info" />
+        </svg>
+        <span>Information</span>
+      </a>
+      <a href="#" class="d-flex mb-400">
+        <svg width="24" height="24" aria-hidden="true" class="mr-400">
+          <use xlink:href="/icons/svg/sprite.svg#link" />
+        </svg>
+        <span>More links</span>
+      </a>
+      <a href="#" class="d-flex mb-400">
+        <svg width="24" height="24" aria-hidden="true" class="mr-400">
+          <use xlink:href="/icons/svg/sprite.svg#profile" />
+        </svg>
+        <span>Profile</span>
+      </a>
+      <a href="#" class="d-flex">
+        <svg width="24" height="24" aria-hidden="true" class="mr-400">
+          <use xlink:href="/icons/svg/sprite.svg#help" />
+        </svg>
+        <span>Help</span>
+      </a>
+    </div>
+  </div>
+</section>
+
+<section class="docs-Section">
+
+  <h2 class="docs-Heading">Usage with offset</h2>
```

Comment:
I guess `.Dropdown--left` was intended here…

Since we simplified the positioning control to a CSS-only solution and moved the responsibility for the correct position to the `DropdownWrapper` (and its styling), I'd delete this example entirely.

![obrazek](https://user-images.githubusercontent.com/5614085/199692906-533c76a8-6d48-40d2-8323-9602ae6c8be8.png)